x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ccr/CCR.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.core.ccr;
+
+public class CCR {
```

Comment:
I think we should stick to conventions for Ccr elsewhere and use `Ccr` as the base name of the class. Also, I would prefer to call it `CcrConstants` just to distinguish it from the other `Ccr` class (which is the plugin, kind of defining the module).